docs/included.md

```diff
@@ -296,7 +368,15 @@
   - 
     [Probability of false detection (WWRP/WGNE Joint Working Group on Forecast Verification Research)](https://www.cawcr.gov.au/projects/verification/#POFD)
 * -  
-    - Sensitivity (Hit Rate, Probability of Detection (POD), True Positive Rate)
+    - Recall (Hit Rate, Probability of Detection (POD), True Positive Rate, Sensitivity)
```

Comment:
I think the "True Positive Rate" is still saying "True Positive Rate, *see Hit Rate*". Is that because there is not an API for True Positive Rate? I asked @tennlee and he suggested perhaps it is worth having an API for True Positive Rate as well.

Reply:
I noticed that - I'll add a function for `true_positive_rate`